lib/models/analytic/aiuta_analytic_event_type.dart

```diff
@@ -0,0 +1,39 @@
+enum AiutaAnalyticEventType {
+  // Pages
+  // Onboarding
+  WelcomePageEntered,
+  HowItWorksPageEntered,
+  BestResultPageEntered,
+  ConsentPageEntered,
+  // Picker
+  ImagePickerPageEntered,
+  LoadingTryOnPageEntered,
+  // Result
+  ResultPageEntered,
+  // History
+  HistoryPageEntered,
+
+  // Actions
+  // Onboarding
+  WelcomeScreenStartButtonClicked,
+  ConsentGiven,
+  // Picker
+  TryOnStarted,
+  NewPhotoStarted,
+  NewPhotoTaken,
+  NewPhotoUploaded,
+  TryOnResultsError,
+  // Result
+  TryOnResultsShown,
+  FlowExit,
+  GeneratedImageShared,
+  ProductLike,
+  ProductAddToCart,
+  // Feedback
+  FeedbackPositive,
+  FeedbackNegative,
+  // History
+  HistoryScreenClicked,
+  GalleryPhotoSelected,
```

Comment:
I don't really get to point of it as
"Click on another photo in scan history to select it for try-on" we don't have this.

Let's remove it, and add OldPhotoSelected near the NewPhotoTaken